modules/nw-ingress-sharding-default.adoc

```diff
@@ -0,0 +1,55 @@
+// Module include in the following assemblies:
+//
+// * ingress-operator.adoc
+// * networking/ingress-sharding.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-sharding-default_{context}"]
+= Sharding the default Ingress Controller
+
+After creating a new Ingress shard, there could be routes that are admitted to your new Ingress shard that are also admitted by the default Ingress Controller. This is because the default Ingress Controller has no selectors and admits all routes by default.
+
+You can restrict an Ingress Controller from servicing routes with specific labels using either namespace selectors or route selectors. The following procedure restricts the default Ingress Controller from servicing your newly sharded `finance`, `ops`, and `dev`, routes using a namespace selector. This adds further isolation to Ingress shards.
+
+[IMPORTANT]
+====
+You must keep all of {product-title}'s administration routes on the same Ingress Controller. Therefore, avoid adding additional selectors to the default Ingress Controller that exclude these essential routes.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in as a project administrator.
+
+.Procedure
+
+. Run the following command to modify the default Ingress Controller:
```

Comment:
Repo guidelines prefer "run the following command" at the end of steps, but I have no idea if it matters much to us.

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output